src/service/performance-impl.js

```diff
@@ -372,7 +383,7 @@ export class Performance {
    * @param {string} label The variable name as it will be reported.
    */
   tickSinceVisible(label) {
-    const now = this.win.performance.now();
+    const now = this.getDomHighResTimeStamp_();
     const visibleTime = this.viewer_ ? this.viewer_.getFirstVisibleTime() : 0;
```

Comment:
We have to change `getFirstVisibleTime` too. I feel like https://github.com/ampproject/amphtml/pull/16119 just needs to be reverted until we redo the review.

Reply:
what is the concern with these values not being normalized, other than extraneous decimal values? Does it warrant a revert rather than just patching it?

Reply:
`performance.now() - Date.now()` gives results like `-1529613244683.8`. This is a p0 regression, since it breaks our perf metrics.

Reply:
Ah I see. I also noticed the bug reporter's comments in the issue. Will revert. Thanks